packages/aws-cdk-lib/aws-s3/lib/notifications-resource/notifications-resource.ts

```diff
@@ -126,6 +127,13 @@ export class BucketNotifications extends Construct {
         }));
       }
 
+      // The customer resource handles unmanaged bucket is the right way so setting the managed flag to false
```

Comment:
This comment is confusing. Can we phrase it better?

Reply:
Done.